### DISCUSSION TITLE
［new ideas］Add ROG peripheral settings

### DISCUSSION THREAD

**Discussion by MTK6891** [seerge/g-helper#2107] [Ideas]
Have you considered adding an option to change the settings of Rog peripherals, such as Rog headset settings😊

**Comment by seerge** [comment]
@MTK6891 check this thread https://github.com/seerge/g-helper/discussions/900

**Reply by MTK6891** [reply]
Hello, I didn’t find my ROG peripherals there. How can I make ghelper support my peripherals?🤔
